molecule/default/tasks/ansiblecredential_test.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Create the tower.ansible.com/v1alpha1.AnsibleCredential
```

Comment:
CI already has coverage here, we can probably remove this file and whatever calls it.